src/scripts/main.js

```diff
@@ -1,3 +1,69 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+const fieldBlock = document.querySelector('.field tbody');
+let field = Array.from(fieldBlock.querySelectorAll('tr'));
+
+document.addEventListener('DOMContentLoaded', () => {
+  fieldBlock.innerHTML = '';
+
+  field.forEach((fiel) => {
+    fieldBlock.appendChild(fiel);
+  });
+
+  updateButtonStates();
+});
+
+function updateButtonStates() {
+  const rowCount = field.length;
+  const columnCount = field.length > 0 ? field[0].children.length : 0;
```

Comment:
Consider checking if `field` is not empty before accessing `field[0].children.length` to avoid potential errors if `field` is empty.